modules/charon-core/src/main/java/org/wso2/charon3/core/extensions/UserManager.java

```diff
@@ -35,6 +35,8 @@
  */
 public interface UserManager {
 
+    public String getContext();
```

Comment:
This will break existing UserManager implementations. Better to define a java8 supported default method.